pkg/duck/resource.go

```diff
@@ -34,82 +34,32 @@ import (
 	"knative.dev/pkg/tracker"
 )
 
-// ResourceInformer is an informer that allows tracking arbitrary Resources.
-type ResourceInformer interface {
```

Comment:
This is used in eventing-contrib. Just a note we'll have to update the code there as well.

Reply:
Thanks for the heads up. I'll update contrib once this is in.